docusaurus.config.js

```diff
@@ -127,11 +127,11 @@ const config = {
         { to: "/blog", label: "Blog", position: "left" },
         { to: "/sponsorship", label: "Sponsor", position: "left" },
         // Right
-        {
-          type: "html",
-          position: "right",
-          value: "(Search coming soon!)",
-        },
+        // {
```

Comment:
I'd rather we just delete. Commented code just always seems like a mistakenly-added addition. When we get search, we'll add it differently (therefore rendering this still dead)